<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewUserDialog</class>
 <widget class="QDialog" name="NewUserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="UserNew">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>744</width>
            <height>594</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QTableView" name="newUserEdit"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Department">
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollArea_4">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>744</width>
            <height>594</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="0" column="0">
            <widget class="QSplitter" name="splitter_6">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <widget class="QSplitter" name="splitter_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <widget class="QRadioButton" name="newDepartment">
               <property name="maximumSize">
                <size>
                 <width>170</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>New Department</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">departments_button_group</string>
               </attribute>
              </widget>
              <widget class="QTableView" name="departmentEdit">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </widget>
             <widget class="QSplitter" name="splitter_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <widget class="QRadioButton" name="existingDepartment">
               <property name="maximumSize">
                <size>
                 <width>170</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Existing Department</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">departments_button_group</string>
               </attribute>
              </widget>
              <widget class="QComboBox" name="departmentExisting">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>35</height>
                </size>
               </property>
              </widget>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Address">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollArea_3">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>744</width>
            <height>594</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_8">
             <item row="0" column="0">
              <widget class="QSplitter" name="splitter_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <widget class="QRadioButton" name="newAddress">
                <property name="maximumSize">
                 <size>
                  <width>120</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>New Address</string>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">address_button_group</string>
                </attribute>
               </widget>
               <widget class="QTableView" name="addressEdit">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QSplitter" name="splitter_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <widget class="QRadioButton" name="existingAddress">
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Existing Address</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">address_button_group</string>
                </attribute>
               </widget>
               <widget class="QComboBox" name="addressExisting">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>30</height>
                 </size>
                </property>
               </widget>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Role">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollArea_2">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>99</width>
            <height>99</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QTableView" name="roleEdit"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="save">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back">
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QPushButton" name="next">
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel</set>
      </property>
      <property name="centerButtons">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewUserDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewUserDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="address_button_group"/>
  <buttongroup name="departments_button_group"/>
 </buttongroups>
</ui>
